Базовая стилизация всех страниц

GALANTfin opened the pull request:
Это выглядит ужасно, я не думал, что это так сложно  :anguished:  , укажи на ошибки и я исправлю :+1:

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2278975/tasks/8)

:boom: https://htmlacademy-htmlcss.github.io/2278975-sedona-36/4/

GALANTfin committed:
Стилизация страниц

Это выглядит ужасно, я не думал, что это так сложно  :anguished:  , укажи на ошибки и я исправлю :+1:

senchkim commented on catalog.html:
```diff
@@ -4,6 +4,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
   <title>Sedona</title>
```
https://validator.w3.org/nu/#textarea
Ошибки в валидаторе на этой странице

senchkim commented on css/style.css:
```diff
@@ -3,8 +3,7 @@ body {
   max-width: 1200px;
```
https://pastenow.ru/da0e3881afd2f9d3e552a7f33b0206b1
Подключил шрифты, названия которых нет в папке `fonts`
Вопрос зачем?

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+@font-face {
+  font-family: "Great Vibes";
+  src: url("../fonts/GreatVibes-Regular.ttf");
+  font-size: normal;
```
нет такого свойства у `font-size`

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.advantages-description {
+  font-family: 'PT Sans', sans-serif\\;
```
зачем на конце 2 слэша?

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.advantages-description-2 {
+  font-family: 'PT Sans', sans-serif\\;
```
зачем на конце 2 слэша?

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.advantages-description-1 {
+  font-family: 'PT Sans', sans-serif\\;
```
зачем на конце 2 слэша?

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.navigation-choise-name {
+  font-family: 'PT Sans';
```
Нужен на конце семейство указывать!

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.navigation-choise-text {
+  font-family: 'PT Sans';
```
Нужен на конце семейство указывать!

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.control-input {
+  font-family: 'PT Sans';
```
Нужен на конце семейство указывать!

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.search {
+  font-family: 'PT Sans';
```
Нужен на конце семейство указывать!
И везде ниже по коду

senchkim commented on css/style.css:
```diff
@@ -47,3 +46,371 @@ body {
+}
+
+.open-page-vector {
+  font-family: 'Great Vibes', sans-serif;
```
Зачем этот шрифт нужен в проекте?

GALANTfin committed:
Исправление ошибок по стилизацири

(+  font-family: 'Great Vibes', sans-serif;

Зачем этот шрифт нужен в проекте?)

Я его добавил, так как не разобрался как сделать такой же шрифт, как на макете иначе.
